### DISCUSSION TITLE
logging house

### DISCUSSION THREAD

**Discussion by xdxxxdx** [Mobility-Data-Space/mobility-data-space#31] [Q&A]
Hello MDS team,
Thanks for sharing the roadmap for 2024.
I see there is a logging house mentioned
![image](https://github.com/Mobility-Data-Space/mobility-data-space/assets/15314807/f85aa043-1c3e-4139-82ee-217298bc9335)
What is the function of this component and its relation to the existing clearing house?
Thanks

**Comment by ip312** [comment]
Hello xdxxxdx. The existing Clearing House is the development from the MDS preliminary project, which no longer meets our current requirements. In addition, the term is now used by Gaia X in a different context. We have therefore named the new implementation Logging House.

**Comment by xdxxxdx** [comment]
Thanks @ip312
